wandb/board/ui/src/components/RunEditor.js

```diff
@@ -46,12 +51,54 @@ class RunEditor extends React.Component {
               name="description"
               rows={12}
               onChange={e =>
-                this.setState({canSubmit: true, content: e.target.value})}
+                this.setState({canSubmit: true, content: e.target.value})
+              }
               placeholder="Provide a description about this project"
               value={this.state.content}
             />
           )}
         </Form.Field>
+        <Form.Group label="Tags">
+          <Form.Input
+            width={3}
+            value={this.state.newTag}
+            onChange={e => {
```

Comment:
You can use (e, {value}) as the argument here. All the Semantic UI component extra the value into that second argument.

Reply:
Fixed